src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/set_global_properties_request.cc

```diff
@@ -151,6 +151,17 @@ void SetGlobalPropertiesRequest::Run() {
     return;
   }
 
+  if (IsCustomizableKeysDisallowed()) {
+    SDL_LOG_ERROR(
+        "Number of customizable keys exceeds the maximum number for this "
+        "layout");
+    SendResponse(
+        false,
+        mobile_apis::Result::INVALID_DATA,
+        "customizeKeys exceeds the number of customizable keys in this Layout");
```

Comment:
```suggestion
        "customizeKeys exceeds the number of customizable keys for current layout");
```

Reply:
@AKalinich-Luxoft this text for message was taken from proposal: 

_If the number of keys in "customizeKeys" array is more than customizable keys allowed, the SDL core should respond with "INVALID_DATA" and the info string should include a detailed message, that "customizeKeys exceeds the number of customizable keys in this Layout"._

Reply:
@ychernysheva I see, that's fine then